mcc/odin/odin/odin_test.go

```diff
@@ -410,6 +410,26 @@ var getCreateDBInstanceInputCases = []getCreateDBInstanceInputCase{
 	},
 }
 
+func equalsCreateDBInstanceInput(input1, input2 *rds.CreateDBInstanceInput) (result bool) {
+	result = true
+	if *input1.AllocatedStorage != *input2.AllocatedStorage {
+		result = false
```

Comment:
go confused me with the result and empty return.
you could also set ```result = false``` first and then return early in each if and set ```result = true``` in the last line.